asmcomp/cmmgen.ml

```diff
@@ -1101,7 +1101,7 @@ struct
   let make_isout h arg =  Cop (Ccmpa Clt, [h ; arg])
   let make_isin h arg =  Cop (Ccmpa Cge, [h ; arg])
   let make_if cond ifso ifnot = Cifthenelse (cond, ifso, ifnot)
-  let make_switch arg cases actions = Cswitch (arg,cases,actions)
+  let make_switch arg cases actions _names = Cswitch (arg,cases,actions)
```

Comment:
if this argument is not needed, why we need this diff

Reply:
to satisfy the module constraint?

Reply:
Yes it's for the module constraint.